tests/dtm/test_multi_m0d_process_restart.py

```diff
@@ -200,10 +200,106 @@ def test_write_during_multi_m0d_restart(self):
         # Assert if process restart failed
         assert_utils.assert_true(resp_proc, "Failure observed during process restart/recovery")
 
-        self.log.info("Step 4: Perform Read Operations on data written in Step 1:")
+        self.log.info("Step 5: Perform Read Operations on data written in Step 1")
         self.dtm_obj.perform_ops(workload_info, que, False, True, True)
         resp = que.get()
         assert_utils.assert_true(resp[0], resp[1])
 
         self.test_completed = True
         self.log.info("ENDED: Verify write during multiple m0d restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-4XXXX")
+    def test_read_during_multi_m0d_restart(self):
+        """Verify read during multiple m0d restart using pkill."""
+        self.log.info("STARTED: Verify read during multiple m0d restart using pkill")
+        log_file_prefix = 'test-4XXXX'
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Create bucket for IO operations")
+        self.s3_test_obj.create_bucket(self.bucket_name)
+
+        self.log.info("Step 2: Perform write Operations")
+        self.dtm_obj.perform_write_op(bucket_prefix=self.bucket_name,
+                                      object_prefix=self.object_name,
+                                      no_of_clients=self.test_cfg['clients'],
+                                      no_of_samples=self.test_cfg['samples'],
+                                      obj_size=self.test_cfg['size'],
+                                      log_file_prefix=log_file_prefix,
+                                      queue=que)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+
+        time.sleep(self.delay)
```

Comment:
maybe just add a logger that u r waiting for xx secs for some ops

Reply:
added